transit_odp/publish/views/timetable/seasonal_services.py

```diff
@@ -177,12 +178,18 @@ def get_form_kwargs(self):
         return kwargs
 
     def get_success_url(self):
+        org_id = self.kwargs["pk1"]
         page = self.request.GET.get("page", None)
+        seasonal_services = SeasonalService.objects.filter(
```

Comment:
You probably want to sort that by end date. In fact this query needs to be the same as the one in the list view.

Reply:
Hey Paul I can do it if you wish, but it's not relevant here the order of the QuerySet I am passing this qs to the `Paginator`